mlrun/projects/pipelines.py

```diff
@@ -912,15 +913,26 @@ def load_and_run(
     engine: str = None,
     local: bool = None,
 ):
-    project = mlrun.load_project(
-        context=f"./{project_name}",
-        url=url,
-        name=project_name,
-        init_git=init_git,
-        subpath=subpath,
-        clone=clone,
-    )
-    context.logger.info(f"Loaded project {project.name} from remote successfully")
+    try:
+        project = mlrun.load_project(
+            context=f"./{project_name}",
+            url=url,
+            name=project_name,
+            init_git=init_git,
+            subpath=subpath,
+            clone=clone,
+        )
+    except Exception as error:
+        # Notifying to slack in case of scheduling:
+        slack = RunNotifications(with_slack=True)
```

Comment:
Please add a TODO to change this to the new notifications once they are merged

Reply:
@quaark once this is merged @yonishelach will do the PR into 1.2.1 with the new mechanism